### PR TITLE
Update to @fastify/send v2

### PR DESCRIPTION
**Pull request by mcollina**
Signed-off-by: Matteo Collina <hello@matteocollina.com>

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -447,12 +450,7 @@ function checkPath (fastify, rootPath) {
 const supportedEncodings = ['br', 'gzip', 'deflate']
 
 function getContentType (path) {
-  const type = send.mime.lookup(path)
-  const charset = send.mime.charsets.lookup(type)
-  if (!charset) {
-    return type
-  }
-  return `${type}; charset=${charset}`
+  return send.mime.getType(path)
```
Comment (Uzlopak) [inline index.js:453]: I think htis has to be:

```suggestion
  const type = send.mime.getType(path)
 
  if (!send.isUtf8Mimetype(type)) {
    return type
  }
  return `${type}; charset=UTF-8`
```

File: test/static.test.js [hunk 2]
```diff
@@ -3619,7 +3619,8 @@ t.test('should serve files into hidden dir without wildcard option', (t) => {
   t.plan(9)
 
   const pluginOptions = {
-    root: path.join(__dirname, '/static-hidden')
+    root: path.join(__dirname, '/static-hidden'),
+    serveDotFiles: true
```
Comment (Uzlopak) [inline test/static.test.js:3623]: I guess this indicates that we break the api contract and have to do a major version bump?

Comment (mcollina) [reply test/static.test.js:3623]: I've fixed that.

**Comment by mcollina** [comment]
@Uzlopak ptal

**Comment by mcollina** [comment]
@Uzlopak unfortunately it's not working

**Comment by Uzlopak** [comment]
fixed the linting issue

**Review by Uzlopak — APPROVED** [review]
LGTM

**Pull request merged**